packages/edit-post/src/components/visual-editor/style.scss

```diff
@@ -28,6 +26,10 @@
 }
 
 .editor-styles-wrapper {
+	// Default background color so that grey .edit-post-editor-regions__content
+	// color doesn't show through.		
+	background-color: $white;
```

Comment:
It seems the background is still valuable if the theme does not provide one, so I moved it to the `.editor-styles-wrapper` (the canvas), which can be be overridden.